### ISSUE TITLE
left alt and meta key not captured

### ISSUE THREAD

**Issue by toqueteos** [skelterjohn/go.wde#26]
Tried with wdetest, my keyboard isn't english and I'm on Win7 x64 maybe that's the problem.

**Comment by sqweek** [comment]
Yep, windows keyboard handling needed some work. I've just updated my fork at http://github.com/sqweek/go.wde - I'd be interested in knowing if the result works sensibly for a non-english keyboard should you still be around. If it doesn't work, best to open an issue there so we don't spam skelterjohn :)
